packages/react/src/ContactPointInput/ContactPointInput.tsx

```diff
@@ -49,25 +57,31 @@ export function ContactPointInput(props: ContactPointInputProps): JSX.Element {
   }
 
   return (
-    <Group spacing="xs" grow noWrap>
+    <Group spacing="xs" grow noWrap align="flex-start">
       <NativeSelect
         data-testid="system"
         defaultValue={contactPoint?.system}
+        required={(systemElement?.min ?? 0) > 0}
```

Comment:
What is the case when `systemElement` is `undefined`?

Not trying to needle.  We have a variety of different philosophies regarding "what should this component do if/when not all data is loaded?"  

1. `return null;`
2. `return <Loader />;`
3. Try your best with what data you do have

That 3rd option gets tricky, because it can lead to lots of content layout shifts and jumpy pages.

If this is by design, that's cool, just want to make sure we're tracking the best option.

Reply:
Good question. This gets at the heart of the new `BackboneElementContext.getNestedElement`. Right now, `getNestedElement`, only returns an element when one has been explicitly defined in the StructureDefinition being displayed. Otherwise, it returns `undefined`; e.g. if you are on the Edit tab instead of the profiles tab. So the idea is that when it is `undefined`, it is safe to assume that the element has the default definition of it's type. Which, in this particular example, [ContactPoint.system is not required](https://hl7.org/fhir/R4/datatypes-definitions.html#ContactPoint.system).

There should not be a "loading" phase here that would lead to  layout shifts since the form isn't rendered until `medplum.requestProfileSchema` or `medplum.requestSchema` returns in the `useEffect` in `ResourceForm.tsx`

Reply:
I was exploring a version where `getNestedElement` would (always) return an element even when it wasn't being modified in some way by a profile, but decided against it since it made `BackbonElementContext` more complicated to implement for seemingly little gain.